src/index.html

```diff
@@ -0,0 +1,79 @@
+        <a href="#">Оплата</a>
+      </div>
+    </header>
+    <main class="content">
+      <section class="left-section">
+        <h3>Полезные ресурсы</h3>
+        <ul class="useful-resources">
+          <li><a href="#">Показы</a></li>
+          <li><a href="#">Статистика продаж</a></li>
+          <li><a href="#">Интервью с руководителями</a></li>
+        </ul>
+      </section>
+      <section class="middle-section">
+        <h1 class="main-title">Автосалон Топкар</h1>
+        <img
+          src="./img/auto.jpg"
+          alt="Car Showroom Topcar image"
+          class="topcar-img"
+        />
+        <p>
+          В автосалоне «Топкар» всегда учитывают индивидуальные пожелания
+          каждого клиента, а проводимые нашим автосалоном акции и специальные
+          предложения делают сотрудничество с нашей компанией выгодным и
+          приятным. На страницах нашего сайта вы можете ознакомиться с
+          предлагаемыми в автосалоне моделями, сравнить технические
+          характеристики различных авто и выбрать наиболее оптимальный вариант.
+          Обращаем ваше внимание, что информация о комплектации, цвете и цене
+          предлагаемых в каталоге авто носит ознакомительный характер, а точные
+          сведения о выбранной модели вы можете получить у наших консультантов
+          по телефону. Покупка автомобиля в нашем салоне станет для вас легким и
+          приятным делом, ведь механизм сделки и порядок оформления
+          документации, включая оформление кредита, отлажен нами до
+          совершенства. В нашем автосалоне вы имеете возможность приобрести
+          автомобили различных марок и комплектаций, начиная от недорогих
+          малолитражек и заканчивая автомобилями мощными внедорожниками с
+          мужским характером. «Топкар» - это автосалон, который постоянно
+          улучшает качество обслуживания и заботится о своих клиентах. Мы
+          стараемся выполнять свои обязанности на самом высоком уровне,
+          предлагая каждому клиенту максимум информации о выбранном автомобиле и
+          подбирая наилучший вариант в соответствии с его пожеланиями. Наши
+          сотрудники будут рады оказать вам всю возможную помощь в выборе и
+          приобретении автомобиля. От нас вы уедете на новом автомобиле с полным
+          баком топлива и регистрацией в ГИБДД. Для гостей столицы из других
+          регионов страны мы также можем сами отравить автомобиль в регион
+          вашего проживания специализированным транспортом.
+        </p>
+      </section>
+      <section class="right-section">
+        <h3>Наши новости</h3>
+        <ul class="news">
+          <li><a href="#">15.12.2023</a></li>
+          <li><a href="#">14.12.2023</a></li>
+          <li><a href="#">13.12.2023</a></li>
+        </ul>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-wrapper">
+        <a href="#">Отзывы</a>
+        <a href="#">Контакты</a>
+        <a href="#">О нас</a>
+      </div>
```

Comment:
Так же как и с header.
```suggestion
  <nav class="footer-wrapper">
    <ul>
      <li><a href="#">Отзывы</a></li>
      <li><a href="#">Контакты</a></li>
      <li><a href="#">О нас</a></li>
    </ul>
  </nav>
```